detox/src/devices/android/ADB.js

```diff
@@ -9,7 +10,14 @@ const Environment = require('../../utils/environment');
 class ADB {
   constructor() {
     this._cachedApiLevels = new Map();
-    this.adbBin = path.join(Environment.getAndroidSDKPath(), 'platform-tools', 'adb');
+    const platformToolsDir = path.join(Environment.getAndroidSDKPath(), 'platform-tools')
+    this.adbBin =
+      which.sync('adb', {path: platformToolsDir, nothrow: true}) ||
+      which.sync('adb', {nothrow: true});
+
+    if (this.adbBin == null){
+      throw new Error(Environment.MISSING_SDK_ERROR)
```

Comment:
I came to the conclusion that you wrote the code you haven't tested manually or via unit tests.
Not good, not good.

You see, `MISSING_SDK_ERROR` is not exported from `./environment.js`, and had you been striving to check all cases manually at least, you would have noticed this issue earlier than me. CI is red now for your pull request, pls fix.

Reply:
I actually did test this code manually, but maybe didn't cover all the code paths.

Reply:
Okay, I see... maybe you haven't added `export ...` to staged changes in Git by an accident and committed without it. Anyways, no offence, man, I'm just concerned about testing edge cases in every piece of functionality that *cannot get a proper unit or e2e test coverage* for some historical or other reasons. As a result, sometimes I don't trust people who claim they have tested their PR thoroughly and if I have spare time I manually check their code on a few different filesystems and OSes. And sometimes those checks turn out to be not so useless. =)